excel/client.bal

```diff
@@ -16,15 +16,21 @@
 
 import ballerina/http;
 
-# Microsoft Excel connector client endpoint
+# Ballerina Microsoft Excel connector provides the capability to access Microsoft Graph Excel API
+# It provides capability to perform perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks]
+# (https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.
+#  
 @display {label: "Microsoft Excel", iconPath: "logo.svg"}
-public client class Client {
-    http:Client excelClient;
-
-    # Initializes the Excel connector client.
-    #
-    # + configuration - Configurations required to initialize the `Client`
-    # + return - An error on failure of initialization else `()`
+public isolated client class Client {
+    final http:Client excelClient;
+
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer 
```

Comment:
To initialize, you can either pass the http:BearerTokenConfig if you have a bearer